utils/single_unit_test_compile.sh

```diff
@@ -29,7 +29,8 @@ export RUNTIME_LIB_DIR=$2
 #aiecc.py --sysroot=$SYSROOT_DIR aie.mlir -I${RUNTIME_LIB_DIR} \
 # ${RUNTIME_LIB_DIR}/test_library.cpp ./test.cpp -o test.elf
 
-aiecc.py --sysroot=$SYSROOT_DIR --aie-generate-xaiev2 ./aie.mlir \
+aiecc.py --sysroot=$SYSROOT_DIR --aie-generate-xaiev2 --no-unified \
+		--host-target=aarch64-linux-gnu ./aie.mlir \
     -DLIBXAIENGINEV2 -I${RUNTIME_LIB_DIR} ${RUNTIME_LIB_DIR}/test_library.cpp \
```

Comment:
Does this mean that --unified is not working?

Reply:
--unified still errors out for xchesscc

Reply:
> --unified still errors out for xchesscc

What's the error?

Reply:
OK, do you have peano and it can't find it, or are you trying to make it work without peano?

Reply:
Work without peano, ensure support for users that only have access to xchesscc

Reply:
got it. I'll take a look at cleaning up the unified flow

Reply:
See https://github.com/Xilinx/mlir-aie/pull/213